testdata/p4tc_samples/add_entry_1_errors.p4

```diff
@@ -49,7 +49,7 @@ struct headers_t {
 struct default_route_drop_params_t {
 }
 
-const ExpireTimeProfileId_t EXPIRE_TIME_PROFILE_NOW    = (ExpireTimeProfileId_t) 2;
+#define EXPIRE_TIME_PROFILE_NOW ((ExpireTimeProfileId_t) 2)
```

Comment:
Is the reason for these changes from const to #define to avoid warnings about unused names declared const, I guess?

If they are truly unused throughout the program, then I suppose that is a reasonable way as any to prevent the warnings.  You could also consider simply deleting the definitions, if they truly are unused in the test program.